app/i18n/locales/vi/what-we-do-section.json

```diff
@@ -0,0 +1,16 @@
+{
+  "whatWeDo": "Chúng tôi làm gì",
+  "whatWeDoDescription": "PSE là một nhóm đa ngành sáng tạo các công cụ mã nguồn mở về quyền riêng tư và mở rộng cho thế hệ internet tiếp theo.",
+  "privacy": {
+    "title": "Quyền riêng tư",
+    "description": "Chúng tôi tin rằng quyền riêng tư là một lợi ích xã hội mà mọi người nên có thể tiếp cận. Đó là lý do tại sao chúng tôi đang tạo ra các công cụ mã nguồn mở giúp mọi người lựa chọn những gì, cách thức, thời điểm và nơi mà họ chia sẻ thông tin."
+  },
+  "scaling": {
+    "title": "Mở rộng",
+    "description": "Hệ thống cơ sở hạ tầng của chúng tôi giúp cộng đồng phát triển bằng cách làm cho Ethereum hiệu quả và dễ tiếp cận hơn. Từ trừng phạt tài khoản và giảm chi phí giao dịch, đến rollups và zkEVM, chúng tôi đang xây dựng hướng tới một tương lai tương thích."
```

Comment:
trừng phạt tài khoản -> tài khoản trừu tượng
Note: can also keep the original technical term Account Abstraction